pad_crm/__manifest__.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2016 Ivan Yelizariev <https://it-projects.info/team/yelizariev>
+# Copyright 2019 Artem Rafailov <https://it-projects.info/team/Ommo73/>
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html)
+{
+    "name": """Pad support""",
+    "summary": """This module adds a Pad to the lead in CRM along with collaborative editing of internal notes and maintains the versions history. Also it automatically increases the left and top indents of the existing text in the modules that use it.""",
+    "category": "Customer Relationship Management",
```

Comment:
It must be one of those values: https://odoo-development.readthedocs.io/en/latest/dev/docs/__manifest__.py.html#category

Why? To be in search results by categories available in apps store
![image](https://user-images.githubusercontent.com/186131/57830730-774bc980-77cc-11e9-813e-e096459dec4c.png)

Reply:
fixed